routes/routes.js

```diff
@@ -9,4 +9,5 @@ const controller = require('../handlers/controller');
 module.exports = (app) => {
     app.route('/ping').get(controller.ping).post(controller.ping);
     app.route('/:id').get(controller.getProduct);
+    app.route('/').post(controller.addProduct);
```

Comment:
Okay. Just remember that this will only get the barcode in the post request, and will have to look for the other info, including the ESG stuff on the server side. 

Also, how will we handle manual entries from the user? Are we going to make a separate post request for that?

Reply:
Manual entries will also be handled via the same routes. We still have few more routes to add, including inserting product to database with unique barcode id.

Reply:
Manual entries will include the following info:
Barcode 
product name
product manufacturer
product category

Can this really work with the same post request?